lib/afterpay/client.rb

```diff
@@ -43,7 +43,7 @@ def default_connection
         Faraday.new(url: self.class.server_url) do |conn|
           conn.use ErrorMiddleware if Afterpay.config.raise_errors
           conn.authorization "Basic", self.class.auth_token
-          # conn.headers["User-Agent"] = Afterpay.config.user_agent_header if Afterpay.config.user_agent_header.present?
+          conn.headers["User-Agent"] = Afterpay.config.user_agent_header if Afterpay.config.user_agent_header.present?
```

Comment:
@sachinsaxena1996 is it better for the caller of the gem to configure individual parameters needed for the user agent and then the gem uses them to construct the user agent header instead of the caller constructing the user agent header?
This way the logic of user agent header construction is abstracted to the gem no?

Reply:
@siddyg then the user have to specify individual values and if it changes at afterpay side will need an update of afterpay-sdk gem.

Reply:
Yes and it should result in afterpay gem version update if afterpay API changes or does a version update

Reply:
Also for these changes in this git repo, should we do a gem version update?

Reply:
@siddyg yes will update version.